src/dev/ci_setup/load_env_keys.sh

```diff
@@ -25,13 +25,13 @@ else
   export VAULT_TOKEN
 
   # Set GITHUB_TOKEN for reporting test failures
-  GITHUB_TOKEN=$(retry 5 vault read -field=github_token secret/kibana-issues/dev/kibanamachine)
+  GITHUB_TOKEN=$(vault_get kibanamachine github_token)
```

Comment:
This doesn't have util.sh sourced.  I don't think we need to worry about this file, it's from jenkins.